examples/visualization/plot_topo_channel_epochs_image.py

```diff
@@ -39,6 +39,9 @@
 picks = mne.pick_types(raw.info, meg='grad', eeg=False, stim=True, eog=True,
                        exclude='bads')
 
+# Plot sensors in 3d
+raw.plot_sensors(kind='3d', ch_type='grad')
```

Comment:
The 3D view defaults to a different orientation than `plot_trans`. Can you make it the same?